src/ACF/Block_Renderer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tribe\Libs\ACF;
+
+class Block_Renderer {
+	public const THEME_PATH_TEMPLATE = 'blocks/%1$s/%1$s.php'; //blocks/hero/hero.php
+
+	public static function render_template( $block, $content, $is_preview, $post_id ) {
+		$name      = str_replace( 'acf/', '', $block[ 'name' ] );
+		$file_path = sprintf( self::THEME_PATH_TEMPLATE, $name );
+
+		$path = locate_template( $file_path );
+
+		if ( ! file_exists( $path ) ) {
+			if ( ! WP_DEBUG ) {
+				return;
+			}
+			echo '<pre>';
+			print_r( $block );
+			print_r( $content );
+			print_r( $is_preview );
+			print_r( $post_id );
+			echo '</pre>';
+
+			return;
+		}
+
+		include( $path );
```

Comment:
Would it be more appropriate to use `get_template_part( $path, null, [ 'block' => $block ] )`? This would allow the block templates to behave like normal WP template parts with respect to accessing global scope, having args explicitly in `$args`, etc.

Reply:
I'm not sure about this but mostly because I may not be fully understanding it. As of right now, we need a `Model` (or another class) to do some data setup/normalization for the component. So `get_template_part( $path, null, [ 'block' => $block ] )` would be doing the same thing as we are doing now right? Let's try to chat about this more today. Certainly open to having more flexibility here.